src/codecs/jats/__file_snapshots__/elife-30274-v1.yaml

```diff
@@ -211,6 +211,9 @@ fundedBy:
       - type: Organization
         name: Laura and John Arnold Foundation
     identifiers: []
+genre:
+  - Replication Study
+  - Cancer Biology
```

Comment:
For the eLife articles it seems that subject areas e.g. `Cancer Biology`, which should be a `DefinedTerm` under `about`, are getting treated as a `genre`. Adding `"heading"` to `ABOUT_TYPES` should resolve this.

```xml
      <article-categories>
        <subj-group subj-group-type="display-channel">
          <subject>Replication Study</subject>
        </subj-group>
        <subj-group subj-group-type="heading">
          <subject>Cancer Biology</subject>
        </subj-group>
      </article-categories>
```